_pages/support.md

```diff
@@ -392,4 +411,60 @@ SAML requests from browser consoles are URI encoded, base-64-encoded, and deflat
 </div>
 </div>
 
+<h4 class="usa-accordion__heading">
+<button class="usa-accordion__button" aria-controls="samlsignaturetroubleshooting">
+SAML Signature Troubleshooting
+</button>
+</h4>
+<div id="samlsignaturetroubleshooting" class="usa-accordion__container">
+<div class="usa-accordion__content" markdown="1"  aria-expanded="true"> 
+Login.gov uses the cryptographic signatures of authentication requests to determine which public certificate to use when encrypting data in the SAML response. If the signature is not present, or cannot be validated successfully, you will encounter problems when you rotate your application’s key pair.
+
+<b> Check signature is present </b>
```

Comment:
[question] should these be headings? they're acting as such, and i think for accessibility purposes it's better for than to be headings